doc/syntax/literals.rdoc

```diff
@@ -198,10 +200,10 @@ a single codepoint in the script encoding:
   ?\C-\M-a #=> "\x81", same as above
   ?あ      #=> "あ"
 
-=== Here Documents (heredocs)
+=== Here Document Literals
 
-If you are writing a large block of text you may use a "here document" or
-"heredoc":
+If you are writing a large block of text you may use a HereDocument,
+sometines written as HereDoc:
```

Comment:
This part should be reverted.  I've not seen anyone refer to heredocs using such capitalization.